lib/public/views/Logs/Logs.js

```diff
@@ -115,7 +120,6 @@ export default class Overview extends Observable {
         const response = await fetchClient('/api/logs', {
             method: 'POST',
             headers: {
-                Accept: 'application/json',
```

Comment:
Any reason for this?

Reply:
@john-ghatas I think this was caused by rebase at some point

Reply:
Done as well